impeller/renderer/backend/vulkan/command_pool_vk.cc

```diff
@@ -52,6 +48,9 @@ std::shared_ptr<CommandPoolVK> CommandPoolVK::GetThreadLocal(
 }
 
 void CommandPoolVK::ClearAllPools(const ContextVK* context) {
+  if (tls_command_pool.get()) {
```

Comment:
Remove this - it's unrelated to the rest of this PR

Reply:
Done, i'll file a new PR.  It was related in that the question was "will we be able to solve the leak once we go down this route, since we know we can with the alternative route".  So the answer is yes.